### PR TITLE
Fix compiler error between FluidNavBarStyle and Diagnosticable which has an empty constructor 

### PR DESCRIPTION
**Pull request by Dorbmon**
Maybe my dart version is to high so that it has this error.

### CODE REVIEW

**Comment by Dorbmon** [comment]
For issue #1

File: lib/src/fluid_nav_bar_style.dart [hunk 1]
```diff
@@ -19,7 +19,7 @@ import 'package:flutter/widgets.dart';
 /// ```
 /// {@end-tool}
 @immutable
-class FluidNavBarStyle extends Diagnosticable {
+class FluidNavBarStyle with Diagnosticable {
```
Comment (eric-taix) [inline lib/src/fluid_nav_bar_style.dart]: As you try to mixin `Diagnosticable`, it's probably better to use directly the mixin, no? 
```suggestion
class FluidNavBarStyle with DiagnosticableMixin {
```

**Comment by eric-taix** [comment]
Close #1

File: lib/src/fluid_nav_bar_style.dart (hunk 1 above)
Comment (Dorbmon) [reply lib/src/fluid_nav_bar_style.dart]: Maybe haha.I am not very familiar with this.

**Review by eric-taix — APPROVED** [review]

**Comment by eric-taix** [comment]
Thanks @Dorbmon for your contribution 🥇
I will make some other minor changes before publishing it to pub.

**Pull request merged**